### PR TITLE
translate(pt): Community page

### PR DESCRIPTION
**Pull request by danilofuchs**
Tracked by https://github.com/microsoft/TypeScript-Website/issues/233

@orta I found some weird keys in english. Could you please take a look?

Original EN file: https://github.com/microsoft/TypeScript-Website/blob/v2/packages//typescriptlang-org/src/copy/en/community.ts

image

@khaosdoctor, @alvarocamillont

### CODE REVIEW

File: packages/typescriptlang-org/src/copy/pt/community.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  com_online_twitter_desc: "Mantenha-se atualizado. Nos siga no Twitter",
+  com_online_blog_desc:
+    "Aprenda sobre os desenvolvimentos mais recentes do Typescript no nosso", //... blog
+  com_online_typed_desc: "Definições de tipos de Typescript.",
+  com_online_typed_href: "Navegue pelas milhares de",
+  com_online_typed_available_for: "bibliotecas e frameworks disponíveis.",
+  com_person: "Conecte-se pessoalmente",
+  com_conferences: "Conferências",
+  com_conferences_alt_img: "logo de ",
+  nav_: {
```
Comment (orta) [inline packages/typescriptlang-org/src/copy/pt/community.ts]: I feel like this might have been me C&Ping

Comment (orta) [reply packages/typescriptlang-org/src/copy/pt/community.ts]: I'll wipe it from everywhere  on the weekend

File: packages/typescriptlang-org/src/copy/en/community.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 export const comCopy = {
-  com_layout_title: "How to set up TypeScript",
+  com_layout_title: "How to set up TypeScript", // FIXME: Is this the right title for the community page?
```
Comment (orta) [inline packages/typescriptlang-org/src/copy/en/community.ts:2]: _might_ be used in the HTML title, but if not yeah, could be a bad C&P

Comment (orta) [reply packages/typescriptlang-org/src/copy/en/community.ts:2]: confirmed:
Screen Shot 2020-07-30 at 5 23 05 PM


It's wrong in English, I think it should probably just be "Connect with other TypeScripters online and offline"

**Comment by khaosdoctor** [comment]
Can we merge this?

**Comment by danilofuchs** [comment]
@khaosdoctor @orta I rebased the branch and fixed the conflicts. Could you guys take a look at it again? Thanks!

**Comment by khaosdoctor** [comment]
lgtm

**Comment by khaosdoctor** [comment]
Hm, this one is out of my scope haha @orta, save us!

**Comment by orta** [comment]
oops, thanks!

**Pull request merged**